clang/lib/Lex/PPDirectives.cpp

```diff
@@ -717,7 +717,13 @@ void Preprocessor::SkipExcludedConditionalBlock(SourceLocation HashTokenLoc,
           assert(CurPPLexer->LexingRawMode && "We have to be skipping here!");
           CurPPLexer->LexingRawMode = false;
           IdentifierInfo *IfNDefMacro = nullptr;
+#ifdef SYCLomatic_CUSTOMIZATION
```

Comment:
Please summary the change to lex in the commit message.

Reply:
done